yt/visualization/tests/test_offaxisprojection.py

```diff
@@ -98,5 +100,4 @@ def test_field_cut_off_axis_octree():
         (p3.frb[("gas", "density")] == p4.frb[("gas", "density")]).all(), False
     )
     p4rho = p4.frb[("gas", "density")]
-    assert_equal(p4rho.min() == 0.0, True)  # Lots of zeros
-    assert_equal(p4rho[p4rho > 0.0].min() >= 0.5, True)
+    assert_equal(np.nanmin(p4rho[p4rho > 0.0]) >= 0.5, True)
```

Comment:
Why not use the following?
```suggestion
    assert np.nanmin(p4rho[p4rho > 0.0]) >= 0.5
```
Note that I am absolutely happy about the PR as is and don't want to delay it any further, so feel free to push the merge button!

Reply:
I think I’m going to keep it as is to match the local dominant convention, even though this is actually not a unit test-based test, so your suggestion should indeed work.